### ISSUE TITLE
[Bug]: "Unable to determine contract standard" using USDC

### ISSUE THREAD

**Issue by AncientSion** [MetaMask/metamask-extension#18111]
### Describe the bug

Trying to SEND UDSC from browser Metamask, i get the "infinite loop" screen. 

Have been trying to hide and reimport, doesnt fix the issue.

The contract in question is the "correct" erc20 USDC

https://etherscan.io/token/0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48?a=0xa0417f82780787251cf216e28e3ccbba49b2765b#code

### Steps to reproduce

1. Try to send USDC
2. Wait

### Error messages or log output

```shell
The console shows:

Error: Unable to determine contract standard
    Error moz-extension://1e87d1ea-cecc-4b7e-bb6a-dd5a128b1019/runtime-lavamoat.js:7522
    getTokenStandardAndDetails moz-extension://1e87d1ea-cecc-4b7e-bb6a-dd5a128b1019/common-4.js:10
    s moz-extension://1e87d1ea-cecc-4b7e-bb6a-dd5a128b1019/common-4.js:10
[sentry-install.js:76:13719](moz-extension://1e87d1ea-cecc-4b7e-bb6a-dd5a128b1019/sentry-install.js)

Error: Unable to determine contract standard
    Error moz-extension://1e87d1ea-cecc-4b7e-bb6a-dd5a128b1019/runtime-lavamoat.js:7522
    getTokenStandardAndDetails moz-extension://1e87d1ea-cecc-4b7e-bb6a-dd5a128b1019/common-4.js:10
    s moz-extension://1e87d1ea-cecc-4b7e-bb6a-dd5a128b1019/common-4.js:10
[sentry-install.js:76:13719](moz-extension://1e87d1ea-cecc-4b7e-bb6a-dd5a128b1019/sentry-install.js)

Uncaught (in promise) Error: Unable to determine contract standard
    Error moz-extension://1e87d1ea-cecc-4b7e-bb6a-dd5a128b1019/runtime-lavamoat.js:7522
    getTokenStandardAndDetails moz-extension://1e87d1ea-cecc-4b7e-bb6a-dd5a128b1019/common-4.js:10
    s moz-extension://1e87d1ea-cecc-4b7e-bb6a-dd5a128b1019/common-4.js:10
[runtime-lavamoat.js:7522:17](moz-extension://1e87d1ea-cecc-4b7e-bb6a-dd5a128b1019/runtime-lavamoat.js)

SES_UNHANDLED_REJECTION: Error: Unable to determine contract standard
    Error moz-extension://1e87d1ea-cecc-4b7e-bb6a-dd5a128b1019/runtime-lavamoat.js:7522
    getTokenStandardAndDetails moz-extension://1e87d1ea-cecc-4b7e-bb6a-dd5a128b1019/common-4.js:10
    s moz-extension://1e87d1ea-cecc-4b7e-bb6a-dd5a128b1019/common-4.js:10
```

### Version

 Version 10.25.0 Last Updated March 6, 2023

### Build type

None

### Browser

Firefox

### Operating system

Windows

**Comment by AncientSion** [comment]
Solved it using this method

https://github.com/MetaMask/metamask-extension/issues/16596

**Comment by bschorchit** [comment]
Could you share more on how you solved using this method? The PR that fixed this issue should have been included already in 10.25. Were you using a production version? Or did you build from develop?

**Comment by seaona** [comment]
I've double-checked on current `develop` and on `10.25.0` production and it's working fine on my end:

https://user-images.githubusercontent.com/54408225/224980269-b30e82f4-d899-4ba3-a67d-416cdcd68a47.mp4
